R/obtain_emiss.R

```diff
@@ -148,15 +152,20 @@ obtain_emiss <- function(object, level = "group", burn_in = NULL){
                                        paste("Category", 1:q_emiss[j])))
         est_emiss[[j]] <- rep(list(est), n_subj)
         names(est_emiss[[j]]) <- paste("Subject", 1:n_subj)
+
+        est_int<-matrix(,ncol = q_emiss[j]-1, nrow = m)
+        emiss_int_est[[j]] <- rep(list(est_int), n_subj)
+        names(emiss_int_est[[j]]) <- paste("Subject", 1:n_subj)
```

Comment:
emiss_int_est is not returned, correct? It's just an internal object in that case, so you do not need to append the subject names to the lists.